docs/release.md

```diff
@@ -0,0 +1,167 @@
+is a logical set of conformance resources, distributed in a package with a
+version that identifies the set of resources and their compatibility characteristics.
+
+The FHIR specification includes a
+[packaging specification](https://confluence.hl7.org/display/FHIR/NPM+Package+Specification)
+which uses NPM packages to distribute the model. When the IG publisher runs
+it produces the NPM package in the user's FHIR package cache at `~/.fhir/packages`
+containing the conformance resources from the FHIR model.
+
+<p align="center">
+<img src="images/npm.png" alt="FHIR NPM" height="200"/>
+</p>
+
+The FHIR packaging specification is still under development and does not have
+a fully fleshed out release process, but we can still use the versioning,
+package naming, etc. guidelines in the specification as a basis for the
+Kids First release specification.
+
+## Versioning
+We will follow the FHIR NPM package specification's versioning scheme which
+uses [Semantic Versioning](https://semver.org/) to determine version numbers
+for FHIR model releases:
+
+<img src="images/semvar.png" alt="Semantic Versioning Example" height="250"/>
+
+### Examples
+
+1. Create first FHIR model release        -> version `0.1.0`
+2. Create new StructureDefinition(s)      -> version `0.2.0`
+3. Remove a StructureDefinition           -> version `1.0.0`
+4. Change StructureDefinition.id          -> version `2.0.0`
+5. Fix typo in StructureDefinition.title  -> version `2.0.1`
+
+## Release Overview
+
+For Kids First, a release of the FHIR model will consist of a little more than
+the NPM package. Here is an overview:
+
+<p align="center">
+<img src="images/release.png" alt="FHIR Model Release"/>
+</p>
+
+Kids First will use Github releases and tags on a `release` branch to manage
+FHIR model releases.
+
+### Testing
+
+The steps in the [Pre-Release](#Pre-Release) and
+[Release Process](#Release-Process) sections include a rigorous testing process
+which includes unit tests and integration tests (read further for details) so
+that when the FHIR model is deployed to the production FHIR server, there is a
+very low chance of failure or issues with the FHIR model resources themselves.
+
+This helps ensure that clients and users of the FHIR services in production
+will have a high quality experience with minimum bugs in the FHIR model.
+
+## Pre-Release
+
+1. Developer creates a feature branch: `add-specimen-conf-resources`
+2. Developer authors the Specimen conformance resources
+   (`StructureDefinition`, `CodeSystem`, `ValueSet`, etc.) and commits locally
+3. Developer runs HL7 FHIR validation on local resources - it passes
+3. Developer pushes changes to remote branch on GitHub
+4. Continuous Integration (CI) pipeline runs:
+    - Unit tests: the same HL7 FHIR validation as in step 3
+    - Integration test(s): load FHIR model resources (conformance and example)
+      into the integration test FHIR server. In the future this may include
```

Comment:
The period in this makes the entry sit awkwardly between being sentenceful and not, because none of the entries end in a period. I suggest making them all end in periods.